flink-libraries/flink-python/src/main/java/org/apache/flink/python/api/functions/PythonMapPartition.java

```diff
@@ -32,7 +32,7 @@
 	private final PythonStreamer streamer;
 	private transient final TypeInformation<OUT> typeInformation;
 
-	public PythonMapPartition(int id, TypeInformation<OUT> typeInformation) {
+	public PythonMapPartition(String id, TypeInformation<OUT> typeInformation) {
```

Comment:
i would propose explicitly passing the envID and setID. Makes it a lot clearer, and we don't have to split a string that we created a second ago.